tests/security/audit/ausearch.pm

```diff
@@ -33,7 +33,8 @@ sub run {
 
     # Extract pid from output log
     script_run("tail -1 $tmp_output > $tmp_backup");
-    script_run("cat $tmp_backup | cut -d '=' -f 9 > $tmp_output");
+    my $cut_index = is_sle('<=12-SP5') ? 7 : 9;
```

Comment:
I'm not sure the 7 and 9 are different here, but I think it's OK after checking.